zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/security/config/ObjectManagementHttpSecurityConfigurer.kt

```diff
@@ -37,6 +37,8 @@ class ObjectManagementHttpSecurityConfigurer : HttpSecurityConfigurer {
                 .hasAuthority(AuthoritiesConstants.ADMIN)
                 .antMatchers(HttpMethod.DELETE, "/api/v1/object/management/configuration/{id}")
                 .hasAuthority(AuthoritiesConstants.ADMIN)
+                .antMatchers(HttpMethod.GET, "/api/v1/object/management/configuration/{configuration_id}/object")
```

Comment:
Why is this `configuration_id` when the url above uses `id`? Also we usually use camel case for these variables.